src/entity-configuration/definitions/fields-defs/experimental.tsx

```diff
@@ -43,7 +44,8 @@ export const FieldsDefinition: FieldsDefinitionRegistry<EntityCoreObjectTypes> =
         return renderEmptyOrValue(
           renderArray(ensureArray({ input: r.species }).map((s) => s.name))
         );
-      else if ('species' in r.subject) return renderEmptyOrValue(r.subject.species.name);
+      else if ('subject' in r && 'species' in r.subject)
```

Comment:
Does it make sense to use optional chaining in such places (I find that way more readable) ? Unless we need to distinguish between falsy values or absent properties (which is not the case here from what I understand).

```suggestion
      else if (r?.subject?.species)
```

Reply:
optional chaining will not fix type issues, the generic type will not cover case by case, so for example `IElectricalCellRecording` will not have subject, so even if you do r.subject you still have type error, but with forcing js checks typescript will infer the rest